RadarSDK/RadarLocationManager.m

```diff
@@ -360,6 +362,19 @@ - (void)updateTracking:(CLLocation *)location
     });
 }
 
+- (void)updateTrackingFromMeta:(RadarMeta *_Nullable)meta {
+    if (meta) {
+        if ([meta trackingOptions]) {
```

Comment:
let's get some logs in both of these conditions to get dashboard visibility into whether remote tracking options have been applied

Reply:
Added.